tests/functional/data/PySide2_QWebEngine/test_web_page.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html lang=en>
+    <head>
+        <meta charset=utf-8>
```

Comment:
Just wondering: Is this relevant? I'm asking since the text contains only ASCII characters. Maybe add some Umlauts?

Reply:
Sure, have added some. I think the utf-8 is a best practice more than anything.